tests/cucumber.conf.js

```diff
@@ -43,6 +46,4 @@ After(async function () {
   await global.context.close();
 });
 
-
-
-module.exports = { tandoorURL, imagePath };
+module.exports = { config };
```

Comment:
export config directly and the you can use it like:
```js
 const config = require('.../cucumber.conf.js')
 // or
 const { tandoorURL } = require('.../cucumber.conf.js')
```
```suggestion
module.exports = config;
```